options/locale/locale_en-US.ini

```diff
@@ -803,8 +802,13 @@ access_token_deletion_cancel_action = Cancel
 access_token_deletion_confirm_action = Delete
 access_token_deletion_desc = Deleting a token will revoke access to your account for applications using it. This cannot be undone. Continue?
 delete_token_success = The token has been deleted. Applications using it no longer have access to your account.
-select_scopes = Select scopes
-scopes_list = Scopes:
+repo_and_org_access = Repository and Organization Access
+permissions_public_only = Public only
+permissions_access_all = All (public, private, and limited)
+select_permissions = Select permissions
+scoped_token_desc = Selected token scopes limit authentication only to the corresponding <a href="/api/swagger" target="_blank">API</a> routes. See <a href="https://docs.gitea.com/development/oauth2-provider#scopes" target="_blank">documentation</a> for more information.
```

Comment:
Move the `<a>` attributes out of translation please. You can use something like this:

```ini
scoped_token_desc = Selected token scopes limit authentication only to the corresponding <a>API</a> routes. See <a>documentation</a> for more information.
```

```handlebars
{{$.locale.Tr "setting.scoped_token_desc" (printf `href="/api/swagger" target="_blank"`) (printf `ref="https://docs.gitea.com/development/oauth2-provider#scopes" target="_blank"`) | Str2html}}
```